Specs/NodeHttpConnector/0.2.3.1/NodeHttpConnector.podspec.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "NodeHttpConnector",
+  "version": "0.2.3.1",
+  "summary": "This coacoapod will encapsulate the logic to communicate to our Node Servers while developing to iOS.",
+  "description": "This pod has the following responsibilities:\n\n+ Abstract http requests\n+ Handle common errors\n+ Support client authentication\n  + No authentication\n  + JWC\n+ Parse http responses",
+  "homepage": "https://github.com/indigotech/br-taqtile-node_http_connector-ios",
+  "license": "MIT",
+  "authors": {
+    "Adriano Valente": "dri9595@gmail.com"
+  },
+  "source": {
```

Comment:
@filipemp i think we could change the `authors` to be something more generic (like `Indigo: indigotech@i.ndigo.com.br` )